group_project/group_project.py

```diff
@@ -306,17 +313,20 @@ def mean(value_array):
             return float(sum(numeric_values)/len(numeric_values))
 
         review_item_data = self.project_api.get_workgroup_review_items_for_group(group_id, self.content_id)
-        review_item_map = {make_key(r['question'], self.real_user_id(r['reviewer'])) : r['answer'] for r in review_item_data}
-        all_reviewer_ids = set([self.real_user_id(r['reviewer']) for r in review_item_data])
-        group_reviewer_ids = [u["id"] for u in self.project_api.get_workgroup_reviewers(group_id)]
-        admin_reviewer_ids = [ar_id for ar_id in all_reviewer_ids if ar_id not in group_reviewer_ids]
+        review_item_map = {
+            make_key(review_item['question'], self.real_user_id(review_item['reviewer'])): review_item['answer']
+            for review_item in review_item_data
+        }
+        all_reviewer_ids = set([self.real_user_id(review_item['reviewer']) for review_item in review_item_data])
+        group_reviewer_ids = [user["id"] for user in self.project_api.get_workgroup_reviewers(group_id)]
+        admin_reviewer_ids = [reviewer_id for reviewer_id in all_reviewer_ids if reviewer_id not in group_reviewer_ids]
```

Comment:
@antoviaque true, but I tried to avoid code changes except renaming loop and list comprehension variables. Since there're almost no test coverage I decided to play safe and avoid any risk of introducing bugs. Going forward this definitely will need some significant improvement.